bestiary/core/errors.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
```

Comment:
Remove this line. This is not a script.

Reply:
I have removed this line from the rest of the tests too.